apps/application-system/form/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,7 @@
 export default {
   production: true,
   baseApiUrl: process.env.API_URL,
+  identityServer: {
```

Comment:
Should the front-end handle the redirection?

Reply:
Hmm I am not sure, I don't see any reason why it should not do so.
If you can think of a reason we should take this up in the `audkenning_og_heimild` slack channel as there are more projects configured this way.